package.json

```diff
@@ -15,7 +15,8 @@
     "format": "turbo run format",
     "test": "turbo run test",
     "createComponent": "node scripts/createComponent.js",
-    "push-docker": "yarn workspace storybook push-docker"
+    "push-docker": "yarn workspace storybook push-docker",
+    "clean": "turbo run clean && rimraf node_modules"
```

Comment:
To confirm, does `turbo` run the command after `run` (in this case `clean`) in all the workspaces/folders as specified in the `workspaces` block on lines 5-8?

![image](https://user-images.githubusercontent.com/45748283/184913588-add85bba-3002-4d2a-acb5-3f2a63c01016.png)

Reply:
yeah, it goes through all the node projects (folder consisting of `package.json` in specified workspaces - level 1 path) and runs the specified command (`clean`) in each of those.
Run locally to confirm ;)